### ISSUE TITLE
secrecy: the trait bound `str: secrecy::CloneableSecret` is not satisfied

### ISSUE THREAD

**Issue by Turbo87** [iqlusioninc/crates#1216]
Up until v0.10 `SecretString` implemented `Clone`, but it looks like this functionality was lost in v0.10. Was that an intentional decision? It does not appear to be mentioned in the changelog.

Related:

- https://github.com/rust-lang/crates.io/pull/9466

**Comment by mempler** [comment]
Weirdly enough, i still have that issue but with serde, even after v1.10.1
```log
the trait `Clone` is not implemented for `str`, which is required by `SecretBox<str>: Deserialize<'_>`

help: the trait `Clone` is implemented for `std::string::String`
note: required for `SecretBox<str>` to implement `Deserialize<'_>`
```

**Comment by tony-iqlusion** [comment]
@mempler fixed in v0.10.2

**Comment by mempler** [comment]
Awesome! Thanks a lot :D
